cmd/cli/kubectl-kyverno/apis/values/values.go

```diff
@@ -1,8 +1,8 @@
 package values
 
 type Values struct {
-	Policies           []Policy            `json:"policies"`
-	GlobalValues       map[string]string   `json:"globalValues"`
-	NamespaceSelectors []NamespaceSelector `json:"namespaceSelector"`
-	Subresources       []Subresource       `json:"subresources"`
+	Policies           []Policy               `json:"policies"`
+	GlobalValues       map[string]interface{} `json:"globalValues"`
```

Comment:
Do we need to cast type here?

https://github.com/kyverno/kyverno/blob/366c687d11fbb8c63994a5d96692ca1f9c463932/cmd/cli/kubectl-kyverno/commands/create/values/command.go#L47

Reply:
Good catch, will check. We definitely need tests for commands now :)

Reply:
But i think we only allow string values in the cli flag 🤔

Reply:
That's probably why we keep it as a string map.

Reply:
The new type handles the string scenario.